src/persist/src/s3.rs

```diff
@@ -89,28 +89,22 @@ impl S3BlobConfig {
     /// `ci/test/cargo-test/mzcompose.yml`.
     ///
     /// For a Materialize developer, to opt in to these tests locally for
-    /// development, use the following values (potentially by putting them in a
+    /// development, follow the AWS access guide:
+    ///
+    ///     https://github.com/MaterializeInc/i2/blob/main/doc/aws-access.md
+    ///
+    /// then use the following values (potentially by putting them in a
     /// shell script and sourcing it if you'll do this often):
     ///
     /// ```shell
-    ///  export MZ_PERSIST_EXTERNAL_STORAGE_TEST_S3_BUCKET="mtlz-test-persist-1d-lifecycle-delete"
-    ///  export AWS_DEFAULT_REGION="us-east-2"
-    ///  export AWS_ACCESS_KEY_ID="<scratch key>""
-    ///  export AWS_SECRET_ACCESS_KEY="<scratch secret>"
-    ///  export AWS_SESSION_TOKEN="<scratch token>"
+    /// export MZ_PERSIST_EXTERNAL_STORAGE_TEST_S3_BUCKET="mtlz-test-persist-1d-lifecycle-delete"
+    /// export AWS_DEFAULT_REGION="us-east-2"
+    /// export AWS_PROFILE="mz-scratch-admin"
+    /// aws sso login
```

Comment:
@danhhz I tested the new instructions and they work great, but wanted to make sure you were on board.